frontend/src/hooks/useRecommitment.tsx

```diff
@@ -55,6 +60,11 @@ export const useRecommitmentCycle = () => {
       recommitmentCycle &&
       offsetTimezoneDate(recommitmentCycle.endDate) >= new Date() &&
       offsetTimezoneDate(recommitmentCycle.startDate) <= new Date(),
+    isRecommitmentReinitiationOpen:
+      recommitmentCycle &&
+      recommitmentCycle.reinitiationEndDate &&
```

Comment:
Do we need to check existence of the re-initiation end date here? I don't think it's hurting anything by including but I was just curious :)

Reply:
Oh oops sorry I just looked at the code - in the diff it looks like this is included in the. 'isRecommitmentCycleOpen' check. Resolving :)